Tools/autotest/arduplane.py

```diff
@@ -1723,6 +1723,41 @@ def MainFlight(self):
         self.run_subtest("Mission test",
                          lambda: self.fly_mission("ap1.txt", strict=False))
 
+    def PitotBlockage(self):
+        '''Test detection and isolation of a blocked pitot tube'''
+        self.set_parameter("ARSPD_OPTIONS",15)
+        self.set_parameter("ARSPD_USE",1)
+        self.set_parameter("SIM_WIND_SPD",10)
+        self.set_parameter("SIM_WIND_DIR",0)
+        self.set_parameter("ARSPD_WIND_MAX",15)
+        self.wait_ready_to_arm()
+        self.change_mode("TAKEOFF")
```

Comment:
```suggestion
        self.change_mode("TAKEOFF")
        self.wait_ready_to_arm()
```

.... assuming we can arm in takeoff :-)

If we were to add a specific check into TAKEOFF which wasn't checked in other modes this would start to fail.